package.json

```diff
@@ -58,7 +58,8 @@
     "compressible": "^2.0.12",
     "duplexify": "^4.0.0",
     "ent": "^2.2.0",
-    "gaxios": "^5.0.0",
+    "extend": "^3.0.2",
```

Comment:
Re-adding extend or merge conflict?

Reply:
Ah conflict, let me get rid of this again.